### PR TITLE
Fixed Issue#2190 | Homepage code syntax error

### PR DESCRIPTION
**Pull request by eshaanmandal**
Fixed issue #2190. Website home page, in the code example 5(Quick & Easy to Learn) has SyntaxError in first line at https://www.python.org/

### CODE REVIEW

**Comment by eshaanmandal** [comment]
Hi,
I think I have fixed all the issues with my last pull request. Please review my changes as a first-time contributor.
Thank you.

**Review by ozgurturkiye — APPROVED** [review]

**Comment by jbw3** [comment]
FYI: I noticed that the next line of code in the example also has an extra backslash.
Instead of this...
```py
name = input('What is your name?\\n')
```
...it should be this...
```py
name = input('What is your name?\n')
```

**Review by ozgurturkiye — APPROVED** [review]

File: codesamples/factories.py [hunk 1]
```diff
@@ -27,11 +27,11 @@ def initial_data():
         (
             """\
             <pre><code><span class=\"comment\"># Simple output (with Unicode)</span>
-            >>> print(\"Hello, I'm Python!\")
+            >>> print("Hello, I'm Python!")
             <span class=\"output\">Hello, I'm Python!</span>
```
Comment (jayaddison) [inline codesamples/factories.py:31]: Shall we put a unicode snake into the code (and HTML)?

```suggestion
            >>> print("Hello, I'm Python \N{SNAKE}")
            <span class=\"output\">Hello, I'm Python! &#x1f40d;</span>
```

image

**Comment by jayaddison** [comment]
@ewdurbin apologies to ping directly, but I thought that this might be worth a quick fixup.  It relates to a syntax error in one of the `python.org` homepage code examples.

@eshaanmandal's fix should work as-is.  I've added a suggestion that we could additionally use unicode in the example code (to match an existing comment), but I could move my suggestion into a separate follow-up pull request.

(also a quick deployment note/reminder: as you [discovered before](https://github.com/python/pythondotorg/pull/1763#issuecomment-1303777674), these examples are fixture data that are loaded into the db to take effect)

**Comment by jayaddison** [comment]
:tada: nice - problem solved :)

image

(confirmed within a Python REPL session too)

**Comment by jayaddison** [comment]
@ewdurbin ek, sorry - a nitpick about this: although the code syntax for this sample has been fixed on the homepage, some of HTML escaping may have gone a bit wonky, and that results in all of the text appearing in green (instead of a mixture of syntax highlighting colours, as with the other samples)

## Expected syntax highlighting in HTML
image

#### HTML snippet
```html
<span class="comment"># Python 3: Simple arithmetic</span>
```

## Appearance of code sample five
image

#### HTML snippet
```html
<span class="\&quot;comment\&quot;"># Simple output (with Unicode)</span>
```

(some kind of HTML-escaping issue?)

**Comment by jayaddison** [comment]
That looks much better now - thank you!

image

**Pull request merged**